src/plugin/cesium/cesiumplugin.js

```diff
@@ -40,6 +40,53 @@ goog.inherits(plugin.cesium.Plugin, os.plugin.AbstractPlugin);
 plugin.cesium.Plugin.ID = 'cesium';
 
 
+/**
+ * @typedef {{
+ *   id: number,
+ *   csPattern: number
+ * }}
+ */
+plugin.cesium.styleLineDashOption;
+
+
+/**
+ * Line dash configurations for Cesium
+ * Patterns based on 16 bit number to make it look consistent between map engines
+ * @type {!Array<!plugin.cesium.styleLineDashOption>}
+ * @const
+ */
+plugin.cesium.LINE_STYLE_OPTIONS = [
+  {
+    id: 0,
+    csPattern: parseInt('1111111111111111', 2) // []
```

Comment:
The default (no dash) pattern should be `undefined`, indicating that a dash material should not be used.